### DISCUSSION TITLE
Issue with starstruck achievement

### DISCUSSION THREAD

**Discussion by mattrighetti** [community/community#20560] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! I've noticed something is not okay with the starstruck achievement, I have 4 folders that have more than 16 stars but my achievement (not really care, actually) is showing a x2 and it shows only a folder on the timeline.

<img width="383" alt="Screenshot 2022-07-14 at 22 46 18" src="https://user-images.githubusercontent.com/16304728/179081199-e7ebd6c2-20a7-44c9-b78c-3ffe19d6d313.png">

**Comment by radar-blip** [comment]
128 stars for bronze (x2) starstruck is correct. Probably only counts the highest number.
You've probably seen this: https://github.com/Schweinepriester/github-profile-achievements#achievements

**Reply by ritul-mtalkz** [reply]
please check this repo and give me star https://github.com/ritulsingh/backend_collection

**Reply by tuves** [reply]
Hi @ritul-mtalkz, we’re glad you’re here 😀

Thanks for sharing your repo with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by D7EAD** [comment, marked as answer]
The achievement will only show the repository that was the first to reach any given milestone. 

For instance, if you got 16 stars in one repository before all others, that would be your first on the timeline. Furthermore, if you were to have a different repository with 128 stars following that, that would be the second listed on the timeline, and so on.

If you have 128 stars on one repository, and that repository was also your first to reach 16 stars, it will be listed twice.

**Reply by ritul-mtalkz** [reply]
please check this repo and give me star https://github.com/ritulsingh/backend_collection

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Comment by joneswilliam1** [comment]
Hey everyone,
**Not an Answer**

This is not an answer but I just got to know about this badge and it seems quite cool I would like to have it, I have a repo which only if you find it to be interesting then go ahead and star it and I'll do the same for you by checking your account and seeing **if** I find something interesting I'll star it too ⭐
https://github.com/abuelgasimsaadeldin/Jetbot-Road-Following-and-Collision-Avoidance

PS. Give me a day, Peace ✌️

**Reply by number0x0001** [reply]
Ok, I starred your repo. Feel free to star mine too: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by joneswilliam1** [reply]
@number0x0001 done

**Reply by notnue** [reply]
Okay done. Have a look at mine too.: https://github.com/notnue/Virtual-Vulnerable-Linux-Machine-for-Penetration-Testing-and-Exploitation.

**Reply by Karkatov** [reply]
Okey 👍👍👍

**Reply by Karkatov** [reply]
> https://github.com/notnue/Virtual-Vulnerable-Linux-Machine-for-Penetration-Testing-and-Exploitation

👍
wait

https://github.com/Karkatov/TaskManager

**Reply by mathcodes** [reply]
shoot while we are at it.... also, take a look and bust out a PR!!! This has potential to really help the community 🦾 [contentfordevelopers](https://github.com/mathcodes/contentfordevelopers/)

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by ritul-mtalkz** [reply]
https://github.com/ritulsingh/Wifi-Password-in-Windows
can any do for me plz

**Reply by michalm57** [reply]
@ritul-mtalkz, @sourabmaity,  @encourage317125 done please star mine too:
https://github.com/michalm57/currency_exchange

**Reply by ritul-mtalkz** [reply]
please check this repo and give me star https://github.com/ritulsingh/backend_collection

**Reply by tuves** [reply]
Hi @joneswilliam1 @sourabmaity @michalm57 @mathcodes @Karkatov @notnue @number0x0001 @encourage317125, we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by AlexRixten** [comment]


**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by Karkatov** [reply]
👍
wait

https://github.com/Karkatov/TaskManager

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by ritul-mtalkz** [reply]
Please leave a star here https://github.com/ritulsingh/Wifi-Password-in-Windows

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Comment by VictorHugoAguilar** [comment]
I need help. Please leave a star here: [https://github.com/VictorHugoAguilar/ticketing-microservices-node-nextjs-react](url).

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by Karkatov** [reply]
👍
wait

https://github.com/Karkatov/TaskManager

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by ritul-mtalkz** [reply]
Please leave a star here https://github.com/ritulsingh/Wifi-Password-in-Windows

**Reply by michalm57** [reply]
@dungng1321 done, Please give me star too :)
https://github.com/michalm57/currency_exchange

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Comment by jfullstackdev** [comment]
just want to comment too, an example is Linus Torvalds, the creator of Linux, it's 100% unlocked already

![image](https://user-images.githubusercontent.com/47092464/181182495-c0ee0094-0207-4eaa-9116-3359f01ffef4.png)

**Reply by ritul-mtalkz** [reply]
please check this repo and give me star https://github.com/ritulsingh/backend_collection

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Reply by diobenu2silva** [reply]
I've done yours.

leave me exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange

**Comment by Karkatov** [comment]
I will mutually add your repository to favorites, here is mine: https://github.com/Karkatov/TaskManager

**Reply by Karkatov** [reply]
> I have added the star ⭐ to you plz gimme :  https://github.com/dev-tom-0108/Lottery_DAO

Done

**Reply by Nayab007** [reply]
i will give you stars give stars as well. thank you

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also @solo0805 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by DevElkami** [reply]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption.

I would appreciate it if you could check it out and give it a star. Thank you.

**Reply by michalm57** [reply]
@DevElkami done please star mine too:
https://github.com/michalm57/currency_exchange

**Comment by Nayab007** [comment]
Can someone give stars to my repos, here is my account www.github.com/nayab007 . I will do the same for your repos if you need stars.

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by DevElkami** [reply]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption.

I would appreciate it if you could check it out and give it a star. Thank you.

**Reply by ritul-mtalkz** [reply]
Please leave a star here https://github.com/ritulsingh/Wifi-Password-in-Windows

**Comment by Nathan-Busse** [comment]
Greetings everyone.

I have been tackling to get this achievement and this badge looks extremelly awesome, I have a repository  which only if you find it to be interesting then go ahead and star it or explore my profile and check out my other projects and I'll do the same for you by checking your project out and staring it in return.

If this was YouTube I would definitely get a strike for illegitamit methods of gaining subscribers:

Nathan-Busse/Minecraft-Bedrock-Server-Control-Panel-GUI: A control dashboard to monitor and control your minecraft bedrock dedicated server through an easy user interface. Created by Nathan-Busse 13 January 2022 Made with Python 3.8 

https://github.com/Nathan-Busse/Minecraft-Bedrock-Server-Control-Panel-GUI


Stay safe everyone.

**Reply by encourage317125** [reply]
hey, I've done yours, so please give me a star.
https://github.com/solo0805/ckeditor5

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Comment by Nayab007** [comment]
Yes I will

**Reply by sourabmaity** [reply]
done yours , star my also 
[(https://github.com/sourabmaity/Vegetable-Detection_App)](https://github.com/sourabmaity/Vegetable-Detection_App)

**Reply by ritul-mtalkz** [reply]
Please leave a star here https://github.com/ritulsingh/Wifi-Password-in-Windows

**Comment by Nayab007** [comment]
Sure I will

**Reply by DevElkami** [reply]
Hi,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption.

I would appreciate it if you could check it out and give it a star. Thank you.

**Comment by Lucocu89** [comment]
> Hello! I've noticed something is not okay with the starstruck achievement, I have 4 folders that have more than 16 stars but my achievement (not really care, actually) is showing a x2 and it shows only a folder on the timeline.
> 
> 
> 
> <img width="383" alt="Screenshot 2022-07-14 at 22 46 18" src="https://user-images.githubusercontent.com/16304728/179081199-e7ebd6c2-20a7-44c9-b78c-3ffe19d6d313.png">
> 
> 
> 
> 

def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls

**Comment by Nayab007** [comment]
Yes same with me, I'm not getting the Starstruck achievement.

**Comment by encourage317125** [comment]
leave it.

**Comment by encourage317125** [comment]
done.

**Comment by DevElkami** [comment]
> Please leave a star here https://github.com/ritulsingh/Wifi-Password-in-Windows

Great, I gave it a star! (Dont forget my project: [Password Vault](https://github.com/DevElkami/PasswordVault) )

**Comment by Nayab007** [comment]
I'll give you star too. Let me get close to my laptop.

**Comment by Nayab007** [comment]
I'll do yours too

**Comment by Nayab007** [comment]
ok done your repos

**Comment by Nayab007** [comment]
I just gave you the Stars

**Comment by Nayab007** [comment]
I have given you some stars

**Comment by diobenu2silva** [comment]
I can exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange

**Reply by Istituto-freudinttheprodev** [reply]
@mattrighetti 
@diobenu2silva 
@Lucocu89 
star mine please I stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Comment by khalifarsm** [comment]
https://github.com/khalifarsm/botox-android

**Comment by kinelho** [comment]
رقم جوال لو سمحت
+966542989102

في السبت، ٢٣ أغسطس ٢٠٢٥، ٢:٢٩ ص Khalifa ***@***.***> كتب:
